src-tauri/src/util.rs

```diff
@@ -143,3 +143,39 @@ pub fn check_northstar_running() -> bool {
         || s.processes_by_name("Titanfall2.exe").next().is_some();
     x
 }
+
+#[allow(dead_code)]
+pub fn copy_dir_all(
+    src: impl AsRef<std::path::Path>,
+    dst: impl AsRef<std::path::Path>,
+) -> std::io::Result<()> {
+    std::fs::create_dir_all(&dst)?;
+    for entry in std::fs::read_dir(src)? {
+        let entry = entry?;
+        let ty = entry.file_type()?;
+        if ty.is_dir() {
+            copy_dir_all(entry.path(), dst.as_ref().join(entry.file_name()))?;
+        } else {
+            std::fs::copy(entry.path(), dst.as_ref().join(entry.file_name()))?;
+        }
+    }
+    Ok(())
+}
+
+pub fn move_dir_all(
```

Comment:
move_dir only moves files, not folders.
So this would result in the folders being left over, but since we delete the whole temp folder it shouldn't matter.

Reply:
(this was also mentioned in the comment)